guides/release/components/triggering-changes-with-actions.md

```diff
@@ -365,37 +370,37 @@ Actions can be invoked on objects other than the component directly from the tem
 import Component from '@ember/component';
 import { inject as service } from '@ember/service';
 
-export default Component.extend({
-  messaging: service(),
+export default class SendMessage extends Component {
+  @service
+  messaging;
 
   // component implementation
-});
+}
 ```
 
 We can tell the action to invoke the `sendMessage` action directly on the messaging service with the `target` attribute.
 
 ```handlebars {data-filename=app/templates/components/send-message.hbs}
 <ButtonWithConfirmation
-    @text="Click to send your message."
-    @onConfirm={{action "sendMessage" "info" target=this.messaging}}
-    as |confirmValue|>
+  @text="Click to send your message."
+  @onConfirm={{action this.messaging.sendMessage "info"}}
```

Comment:
@pzuraq is this correct, nothing special is required any more?

Reply:
`sendMessage` needs to be decorated with `@action`, if it is then this should work